src/components/SongCard/SongCard.css

```diff
@@ -1,7 +1,7 @@
 .song-card {
   border: 2px solid black;
   display: grid;
-  position: relative;
+  /* position: relative; */
```

Comment:
this line is important to the absolute positioning of the buttons on each card, if we want them to float to the right I wouldn't recommend removing this

Reply:
ahh, sorry, I commented out because I wasn't sure what it was doing and forgot to comment it back in. I was thinking maybe we could utilize flexbox for the card? I wasn't sure what your ultimate layout goal was for the card.